src/main/java/az/gdg/msauth/model/dto/MailDTO.java

```diff
@@ -0,0 +1,19 @@
+package az.gdg.msauth.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class MailDTO {
+
+    private List<String> mailTo;
+    private String mailSubject;
+    private String mailBody;
+}
```

Comment:
why in MailDTO you have to specify mailTo, mailSubject, mailBody ? Isn't **to**, **subject** and **body** enough ?